include/tester/vtcp/VTCPRequestHandler.hpp

```diff
@@ -49,7 +49,7 @@ class VTCPRequestHandler : public RequestHandler
     {
         std::cout << "Client disconnected" << std::endl;
 
-        //sender->onDisconnect();
+        //sender->disconnect();
```

Comment:
Either remove the comment or actually use this piece of code to test if disconnect works as it should.

Reply:
done